event_routing_backends/models.py

```diff
@@ -83,11 +160,11 @@ class RouterConfiguration(ConfigurationModel):
     .. no_pii:
 
     """
-
-    KEY_FIELDS = ('backend_name',)
+    KEY_FIELDS = ('route_url',)
+    enabled = models.BooleanField(default=False, verbose_name=_("Enabled"))
     backend_name = models.CharField(
```

Comment:
we should place an index on `backend_name` since it is no more in `KEY_FIELDS` now.

Reply:
Done

Reply:
@rehanedly I can't see index being added on the field.

Reply:
Meta:
     indexes = [models.Index(fields=['enabled', 'backend_name'])]
     
I have made index using this method

Reply:
@rehanedly actually we don't need `enabled` fields it is already in [ConfigurationModel](https://github.com/edx/django-config-models/blob/master/config_models/models.py#L94)

Reply:
enabled field in ConfigurationModel is not index. Should we make that index?